grails-app/controllers/org/pih/warehouse/api/SelectOptionsApiController.groovy

```diff
@@ -45,8 +45,11 @@ class SelectOptionsApiController {
     }
 
     def catalogOptions() {
-        List<ProductCatalog> catalogs = genericApiService.getList(ProductCatalog.class.simpleName, [sort: "name"]).collect {
-            [id: it.id, label: "${it.name} (${it?.productCatalogItems?.size()})"]
+        boolean hideNumbers = params.boolean('hideNumbers', false)
```

Comment:
nitpicky: we tend to use `" "` on the backend and it feels like you changed them from `" "` to `' '`